### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [dangokanako/blog-comments-gitalk#22] [Announcements]
# about

关于博主是笨蛋。 不值一提的人，没什么了解的必要。 博客搭建喜欢也可以自己搭一个，全免费。但如果部署在github pages的话，会像访问这个网站一样难受 教程供参考https://dangokanako.github.io/2023/04/29/%E6%8A%80%E6%9C%AF/%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA/ 开发计划从1开始的前端生活做了两个小

https://dangokanako.github.io/about.html

**Comment by Endless11akaLin** [comment]
发现小鸟秘密基地(*^▽^*)

**Comment by yumenomajo** [comment]
有趣的小站。ヽ( ^ω^ ゞ )

**Comment by Sill-William** [comment]
鸟叔之爱，宜乎众矣

**Comment by CaTechX** [comment]
非常喜欢您的文字
